mobile-browser-based-version/src/components/MNISTComponents/MNIST_script.js

```diff
@@ -0,0 +1,146 @@
+import * as tf from "@tensorflow/tfjs";
+
+
+
+/**
+ * Object used to contain information about the task in general, the model's limitations
+ * and the data accepted by the model 
+ */
+export const display_informations = {
+    // {String} title of the task (keep it short ex: Titanic)
+    taskTitle: "MNIST",
+    // {String} simple overview of the task (i.e what is the goal of the model? Why its usefull ...)
+    overview: "The MNIST handwritten digit classification problem is a standard dataset used in computer vision and deep learning. Although the dataset is effectively solved, we use it to test our Decentralised Learning algorithms and platform.",
+    // {String} potential limitations of the model 
+    limitations: "The current model is a very simple CNN and its main goal is to test the app and the Decentralizsed Learning functionality.",
+    // {String} trade-offs of the model 
+    tradeoffs: "We are using a simple model, first a 2d convolutional layer > max pooling > 2d convolutional layer > max pooling > convolutional layer > 2 dense layers.",
+    // {String} information about expected data 
+    dataFormatInformation: 
+    "This model takes as input an image dataset. You can upload each digit image of your dataset in the box corresponding to its label. The size of the image will be reduced to 28x28. Finally, the image should be in grey scale.",
+    // {String} description of the datapoint given as example
+    dataExampleText: "Below you can find an example of an expected image representing the digit 9.",
+};
+
+/**
+ * Object used to contain the model's training specifications
+ */
+export const training_information = {
+    // {String} model's identification name
+    model_id: "mnist-model",
```

Comment:
We can adopt the following rule: before handling a routing path, we lower case the task's name. This would mean that all paths in the router should be lowered case. 
This will help us modularize without errors the routing of tasks in each component.

Reply:
That seems like a good idea to me. But it is what we are already doing right?